src/main/java/io/pakland/mdas/githubstats/domain/model/HistoricQueries.java

```diff
@@ -0,0 +1,25 @@
+package io.pakland.mdas.githubstats.domain.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "historic_queries")
+public class HistoricQueries {
+    @Id
+    @Column(name = "id", updatable = false, nullable = false)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne()
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```

JPA by default uses eager fetch, which is un-optimal because didn't want to retrieve until needed.

Reply:
Understood